topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,718 @@
+>
+> 3. **IGV** {% icon tool %} to inspect with the signal coverage and log2 ratio tracks
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+> 
+> 1. How many peaks have been identified in `chrX:151,385,260-152,426,526` based on IGV?
+>
+>    ![Output of MACS2](../../images/formation_of_super-structures_on_xi/macs2_igv.png "Peaks for wt_H3K4me3_rep1 and wt_input_rep1 on chrX:151,385,260-152,426,526")
+>
+> 2. What are the fold change of the peaks identified in `chrX:151,385,260-152,426,526`? Hint: using the BED file
+> 3. How many peaks have been identified on the full chromosome X? How many peaks have a fold change > 50?
+>
+> > ### {% icon solution %} Solution
+> > 1. We can see 11 peaks (track below the genes). 
+> > 2. Using **Filter** {% icon tool %} with `c2>151385260 and c3<152426526`, we found that the 11 peaks have a fold change 3.81927 and 162.06572
+> > 4. On the 656 peaks on the full chromosome (number of lines of the original BED file) there are 252 peaks with FC>50 (using **Filter** {% icon tool %} with `c7>50`)
+> {: .solution } 
+{: .question}
+
+The called peak regions can be filtered by, *e.g.* fold change, FDR and region length for further downstream analysis.
+
+# Step 6: Plot the signal between samples
+
+We normalized our data and identied the peaks. We would like now to visualize scores associated with genomic regions, for example ChIP enrichment values around the TSS of genes. And we would like to compare it with another sample (CTCF).
+
+Since we already generated the required files for the H3K4me3 sample, let's make them only for the CTCF sample:
+
+> ### {% icon hands_on %} Hands-on: Prepare the peaks and data for CTCF
+> 
+> 1. **bamCompare** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"First BAM file (e.g. treated sample)"*: `wt_CTCF_rep1.bam`
+>    - {% icon param-file %} *"Second BAM file (e.g. control sample)"*: `wt_input_rep1.bam`
+>    - *"Bin size in bases"*: `50`
+>    - *"How to compare the two files"*: `Compute log2 of the number of reads ratio`
+>    - *"Coverage file format"*: `bigwig`
+>    - *"Region of the genome to limit the operation to"*: `chrX`
+> 2. Rename the output of **bamCompare** {% icon tool %} with the name of the sample
+> 3. **MACS2 callpeak** {% icon tool %} with the following parameters
+>    - *"Are you pooling Treatment Files?"*: `No`
+>       - {% icon param-file %} *"ChIP-Seq Treatment File"*: `wt_CTCF_rep1.bam`
+>    - *"Do you have a Control File?"*: `Yes`
+>       - *"Are you pooling Treatment Files?"*: `No`
+>           - {% icon param-file %} *"ChIP-Seq Treatment File"*: `wt_input_rep1.bam`
+>    - *"Format of Input Files"*: `Paired-end BAM`
+>    - *"Effective genome size"*: `M.musculus (1.87e9)`
+>
+{: .hands_on}
+
+We can now concatenate the MACS2 outputs with the location of the peaks (concatenate the files and merge the overlapping regions) to obtain one BED file corresponding to the coordinates of the interesting regions to plot.
```

Comment:
Why do you concatenate it? Is there any benefit beside having just one file in the end?